.drone.star

```diff
@@ -146,7 +146,7 @@ def main(ctx):
 
     build_release_helpers = [
         changelog(ctx),
-        docs(ctx),
+        # docs(ctx),
```

Comment:
we need to add this again. Just empty the `config-docs-generate` target in `.make/docs.mk`